client/templates/userprofile/settings.js

```diff
@@ -54,16 +55,43 @@ Template.profileSettings.events({
       {
           Session.set("realerror", "Your Realname is empty!");
       }
+      setTimeout(function() {
```

Comment:
Use reactive variables instead of the global Session variable. I 
outlined some resources in the pull request discussion.

On 13.09.2015 20:35, sbe wrote:
>
> In client/templates/userprofile/settings.js 
> <https://github.com/Crowducate/crowducate-platform/pull/116#discussion_r39349883>:
>
> > @@ -54,16 +55,43 @@ Template.profileSettings.events({
> >        {
> >            Session.set("realerror", "Your Realname is empty!");
> >        }
> > +      setTimeout(function() {
>
> I'm open for any suggesting to achieve the same.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Crowducate/crowducate-platform/pull/116/files#r39349883>.
>